### DISCUSSION TITLE
[Feature] Can skywalking collect input parameters and output result parameters of the microservice api?

### DISCUSSION THREAD

**Discussion by fisea001** [apache/skywalking#9204] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

I am a .net developer,using skywalking 8.6 , In order to solve problem quickly,I want to collect  input parameters and output result parameters of the microservice api.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
Dot net agent is not official in the Apache Software Foundation, you submitted the issue in the wrong place.

**Reply by fisea001** [reply]
> Dot net agent is not official in the Apache Software Foundation, you submitted the issue in the wrong place.

Hi,Where I should submit the issue? thanks.

**Reply by Superskyyy** [reply]
> > Dot net agent is not official in the Apache Software Foundation, you submitted the issue in the wrong place.
> 
> Hi,Where I should submit the issue? thanks.

https://github.com/SkyAPM/SkyAPM-dotnet/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc Here :)
